src/components/input/select/index.tsx

```diff
@@ -163,7 +171,14 @@ export class SmoothlyInputSelect implements Input, Editable, Clearable, Componen
 		!this.readonly && (this.open = !this.open)
 		this.filter = ""
 	}
-
+	areValuesEqual(selected: HTMLSmoothlyItemElement[], initialValue: HTMLSmoothlyItemElement[]): boolean {
+		return selected.length === initialValue.length && initialValue.every(value => selected.includes(value))
+	}
+	displaySelected(): void {
+		const displayString: string = this.selected.map(option => option.innerHTML).join("    ")
```

Comment:
instead of joining with a magical number of spaces we should probably join with an empty string and use CSS to style this. for example make the "main element" (a variable that should probably change name now that it is no longer a main element) display: flex with some gap.